Address ESLint warnings about clickable image in documentation_node.js

```
/Users/JM3312/code/eto/eto-supply-chain/supply-chain/src/components/documentation_node.js
  119:15  warning  Visible, non-interactive elements with click handlers must have at least one keyboard listener  jsx-a11y/click-events-have-key-events
  119:15  warning  Non-interactive elements should not be assigned mouse or keyboard event listeners               jsx-a11y/no-noninteractive-element-interactions
```

--- a/supply-chain/src/components/documentation_node.js
+++ b/supply-chain/src/components/documentation_node.js
@@ -151,7 +151,7 @@ const DocumentationNode = (props) => {
                 }}
                 role="presentation"
                 tabIndex={0}
-                alt={`Image of ${meta.image_caption}`}
+                alt={meta.image_caption}
               />
             </div>
           }

